Dockerfile

```diff
@@ -5,13 +5,25 @@ ENV BAMBOO_HOME     /var/atlassian/bamboo
 ENV BAMBOO_INSTALL  /opt/atlassian/bamboo
 ENV BAMBOO_VERSION  6.0.0
 
+# Copy in the gpg key for github_git-lfs so that we can load git-lfs
+# into the image.
+COPY github-lfs-repo.gpgkey /tmp
+
 # Install Atlassian Bamboo and helper tools and setup initial home
 # directory structure.
 RUN set -x \
+    && echo 'Adding repository and required key to install git-lfs' \
+    && apt-key add /tmp/github-lfs-repo.gpgkey \
```

Comment:
I don't like just pulling in random files from the net without some verification that I am getting a verified version of it.

The other way to do it that I have seen is to pull in the key from the central key servers, so something like (taken from the sonar-scanner image):

# pub   2048R/D26468DE 2015-05-25
    #       Key fingerprint = F118 2E81 C792 9289 21DB  CAB4 CFCA 4A29 D264 68DE
    # uid                  sonarsource_deployer (Sonarsource Deployer) <infra@sonarsource.com>
    # sub   2048R/06855C1D 2015-05-25
    && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys F1182E81C792928921DBCAB4CFCA4A29D26468DE \